integration-cli/docker_api_containers_test.go

```diff
@@ -1403,18 +1403,6 @@ func (s *DockerSuite) TestPostContainersCreateShmSizeNegative(c *check.C) {
 	c.Assert(string(body), checker.Contains, "SHM size must be greater then 0")
 }
 
-func (s *DockerSuite) TestPostContainersCreateShmSizeZero(c *check.C) {
```

Comment:
this test cannot be removed. `ShmSize = 0` is a valid value that you're ignoring because of the pointer change in host config.